lib/src/insertion_order_parser.dart

```diff
@@ -0,0 +1,199 @@
+import 'dart:convert';
+
+import 'proto/insertion_order.pb.dart';
+
+class InsertionOrderParser {
+  static const _emptyEntry = '';
+
+  static final _entityStatusMap = {
+    for (var status in InsertionOrder_EntityStatus.values) status.name: status
+  };
+
+  static final _pacingPeriodMap = {
```

Comment:
nice use of language features

Reply:
I think it's better to ensure the map is actually typed:
  static final PACING_PERIOD_MAP = Map<String, InsertionOrder_Pacing_PacingPeriod> 
     .fromIterable(InsertionOrder_Pacing_PacingPeriod.values, k -> k.name, v -> v);
If you want to stick with your current syntax, make sure to use 'final' instead of 'var' in the for loops.